src/operator/tensor/elemwise_binary_op.h

```diff
@@ -525,6 +525,67 @@ class ElemwiseBinaryOp : public OpBase {
     });
   }
 
+  template<typename xpu, typename OP>
+  static void MixedUnaryBackwardUseInCompute(const nnvm::NodeAttrs &attrs,
+                                             const OpContext &ctx,
+                                             const std::vector<TBlob> &inputs,
+                                             const std::vector<OpReqType> &req,
+                                             const std::vector<TBlob> &outputs) {
+    using namespace mxnet_op;
+    if (req[0] == kNullOp) return;
+    Stream<xpu> *s = ctx.get_stream<xpu>();
+    CHECK_EQ(inputs.size(), 2U);
+    CHECK_EQ(outputs.size(), 1U);
+    if (!mxnet::common::is_float(inputs[1].type_flag_)) {
+      LOG(FATAL) << "Operator " << attrs.op->name <<
+                    " does not support type " << inputs[1].type_flag_;
+    }
+    if (outputs[0].type_flag_ == mshadow::kBool) {
+      LOG(FATAL) << "Operator " << attrs.op->name << " does not support boolean type";
```

Comment:
Same for the error message here since `outputs[0].type_flag_` is technically same as `inputs[1].type_flag_`